src/ai/backend/manager/models/audit_logs.py

```diff
@@ -0,0 +1,240 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Optional, Sequence
+
+import graphene
+import sqlalchemy as sa
+from dateutil.parser import parse as dtparse
+from graphene.types.datetime import DateTime as GQLDateTime
+from sqlalchemy.dialects import postgresql as pgsql
+from sqlalchemy.engine.row import Row
+
+from ai.backend.common.logging import BraceStyleAdapter
+
+from .base import Item, PaginatedList, metadata, simple_db_mutate
+from .minilang.ordering import QueryOrderParser
+from .minilang.queryfilter import QueryFilterParser
+from .user import UserRole
+
+if TYPE_CHECKING:
+    from .gql import GraphQueryContext
+log = BraceStyleAdapter(logging.getLogger(__name__))
+__all__: Sequence[str] = (
+    "audit_logs",
+    "AuditLog",
+    "AuditLogList",
+    "AuditLogInput",
+    "CreateAuditLog",
+)
+
+audit_logs = sa.Table(
+    "audit_logs",
+    metadata,
+    sa.Column("user_id", sa.String(length=256), index=True),
+    sa.Column("access_key", sa.String(length=20), index=True),
+    sa.Column("email", sa.String(length=64), index=True),
+    sa.Column(
+        "action",
+        sa.Enum("CREATE", "CHANGE", "DELETE", name="auditlogs_action", create_type=False),
```

Comment:
Please define `action` enum as python enum type, so that we can handle these values more pythonic- way. Ref: https://github.com/lablup/backend.ai/blob/main/src/ai/backend/manager/models/user.py#L74